packages/protoc-gen-es/README.md

```diff
@@ -31,45 +24,92 @@ pete = Person.fromBinary(bytes);
 pete = Person.fromJsonString('{"name": "pete", "id": 123}');
 ```
 
-Learn more at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
+Learn more about the project at [github.com/bufbuild/protobuf-es](https://github.com/bufbuild/protobuf-es).
 
 
 ## Installation
 
-### With npm
+`protoc-gen-es` is a code generator plugin for Protocol Buffer compilers,
+like [buf](https://github.com/bufbuild/buf) and [protoc](https://github.com/protocolbuffers/protobuf/releases).
+It generates base types - messages and enumerations - from your Protocol Buffer 
+schema. The generated requires the runtime library [@bufbuild/protobuf](https://www.npmjs.com/package/@bufbuild/protobuf).
+
+To install the plugin and the runtime library, run:
 
 ```shell
-npm install @bufbuild/protoc-gen-es
+npm install --save-dev @bufbuild/protoc-gen-es
+npm install @bufbuild/protobuf
```

Comment:
With npm, it isn't necessary to install the runtime library explicitly. It is a peer dependency of the plugin, so npm will install it automatically. But yarn and pnpm don't, so it seems a bit clearer to add it in this example anyways.